### PR TITLE
glide 0.6.0

### PR DESCRIPTION
**Pull request by mattfarina**

### CODE REVIEW

File: Library/Formula/glide.rb [hunk 1]
```diff
@@ -3,15 +3,8 @@
 class Glide < Formula
   desc "Simplified Go project management, dependency management, and vendoring"
   homepage "https://github.com/Masterminds/glide"
-  url "https://github.com/Masterminds/glide/archive/0.5.1.tar.gz"
-  sha256 "b7dd696690cee45f505cfab9652602208d6da6e5453213a5ac14b82d1b968628"
-
-  bottle do
```
Comment (dunn) [inline Library/Formula/glide.rb]: Leave the bottle block; our bot takes care of it!

**Comment by DomT4** [comment]
@BrewTestBot test this please

**Comment by mattfarina** [comment]
Glide 0.6.1 is about to come out. A couple bugs were found and fixed over the weekend. I'll create a new pull request for that when it's ready.

**Comment by DomT4** [comment]
Thanks @mattfarina. Apologies for the delay here; we've had a lot of CI issues over the last ~10 days and have only managed to get on top of the backlog properly starting yesterday.

**Pull request closed without merging**